examples/tictactoe/src/server.ts

```diff
@@ -69,7 +69,7 @@ export class TicTacToe extends Agent<Env, TicTacToeState> {
 
     // now use AI to make a move
     const { object } = await generateObject({
-      model: openai("gpt-4o"),
+      model: openai("gpt-4o") as any,
```

Comment:
having to do as any is usually a bad sign. especially when this is supposed to be a "native" arg